FreeRTOS_UDP_IP.c

```diff
@@ -120,6 +120,16 @@ static eARPLookupResult_t prvLookupIPInCache( NetworkBufferDescriptor_t * const
                 }
             }
         #endif
+        #if ( ipconfigUSE_MDNS == 1 )
+            {
+                /* mDNS messages are typically used on a LAN and they're
```

Comment:
I am not sure how this comment relates to the code below. Can you explain?

Reply:
Good catch, the entire code was:
~~~c
    /* mDNS messages are typically used on a LAN and they're
     * not supposed to cross routers */
    if( pxNetworkBuffer->ulIPAddress == ipMDNS_IP_ADDRESS )
    {
        pxUDPPacket->xIPHeader.ucTimeToLive = 0xffU;
    }
~~~
The comment contradicts with the code.
The [recommended value is 255](https://datatracker.ietf.org/doc/html/rfc6762#section-11), so I will change the comment to:
~~~c
/* mDNS messages have a hop-count of 255, see RFC 6762, section 11. */
~~~